src/components/PrivateRoute.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
Do you think Authenticated is a bettter name than Private?  I worked on another project where they used PrivateRoute and PublicRoute and I always had to think about the meaning.  Public would be Unauthenticated.